### DISCUSSION TITLE
Cannot correct my EDU email address

### DISCUSSION THREAD

**Discussion by hshsilver** [community/community#83023] [GitHub Education]
### Hi everyone,

Hey guys. Does anyone know how to correct the EDU email address? 
I have a previous edu email address "A" from my previous college, and now I re-verify the student pack with documents with a new edu email address "B". 
I just realized the account default edu email is still connected to the previous one "A".
I tried to choose the new one "B" but could not click the CONTINUE button because it's "grey".

<img width="734" alt="image" src="https://github.com/community/community/assets/25773429/da2bb256-ff0f-4e05-b2b5-ddc2ddd4dacd">

Thx!

**Comment by Ashishdheer** [comment]
Hey @hshsilver  if you're having trouble updating your EDU email address, try clearing your browser's cache and cookies or using a different browser. Make sure the information you entered is accurate, and check if there are any browser extensions causing issues. If the "CONTINUE" button is still grayed out, contact the customer support of the service for assistance. I think waiting for a while and retrying the process might also help, and if all else fails, the support team should be able to guide you through resolving the issue.

**Reply by hshsilver** [reply]
Hi! I tried all the methods you mentioned and still cannot change/choose the email address to the new one. Do you know how to contact the support team? Thx

**Reply by Ashishdheer** [reply]
To contact the support team, first, visit the official website of the service and look for a "Contact Us" or "Support" section. Check for options like live chat, email support, or a support ticket system. You can also find a customer support email address or phone number on their website.
